scripts/osfstorage/glacier_audit.py

```diff
@@ -66,11 +66,10 @@ def get_job(vault, job_id=None):
 
 
 def get_targets(date):
-    return FileVersion.find(
-        Q('date_created', 'lt', date - DELTA_DATE) &
-        Q('status', 'ne', 'cached') &
-        Q('metadata.archive', 'exists', True) &
-        Q('location', 'ne', None)
+    return FileVersion.objects.filter(
```

Comment:
I'm not certain, but I think `metadata__archive__isnull=False` should be `metadata__has_key='archive'` instead.

https://docs.djangoproject.com/en/1.11/ref/contrib/postgres/fields/#has-key